concordium-consensus/src/Concordium/KonsensusV1/Types.hs

```diff
@@ -431,6 +448,8 @@ data TimeoutMessageBody = TimeoutMessageBody
       tmFinalizerIndex :: !FinalizerIndex,
       -- |Round number of the round being timed-out.
       tmRound :: !Round,
+      -- |Epoch of which the round being timed-out.
+      tmEpoch :: !Epoch,
```

Comment:
Same here. Remove `tmEpoch`.

Reply:
Addressed this, please have a look and confirm that things are as expected.